src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -2661,6 +2661,507 @@ Note: `aparse()` is case-insensitive.
     </CollapserGroup>
   </Collapser>
 
+  <Collapser
```

Comment:
Functions are in alphabetical order; this should go down between substring() and upper().

Reply:
Updated the order of the `toTimestamp()` function